src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/ServiceAgencySwitchResourceIntTest.kt

```diff
@@ -33,13 +33,14 @@ class ServiceAgencySwitchResourceIntTest : ResourceTest() {
 
   private lateinit var leeds: AgencyLocation
   private lateinit var moorland: AgencyLocation
+  private lateinit var someService: ExternalService
```

Comment:
very confusing here - `ExternalService` isn't a service 🤯

Reply:
Hmmm. The table is called `EXTERNAL_SERVICE`. Should I call the class `ExternalServiceTable`?